src/core/schedule.js

```diff
@@ -25,21 +16,12 @@ export const getScheduleOutput = (lessonType, { start, end, title, description,
 	return `\n\n⏰ ${startTime} - ${endTime}\n${lessonType}:\n<b>${lessonTitle}</b>\n${lessonDescription}\n${audienceInfo}`
 }
 
-/**
- * Retrieves information about the lessons for the specified day.
- *
- * @param {string} targetDay - The target day for which lessons information is required.
- * @param {Array} scheduleData - The array of objects containing schedule data.
- * @param {boolean} shouldShiftWeek - Flag defining today is Sunday and the user has clicked the 'Schedule for tomorrow' button.
- * @param {number} weekNumber - The current week number.
- * @returns {string} - Formatted lessons information or NO_LESSONS_MESSAGE if no lessons found.
- */
-
-export function getLessonsForDay(targetDay, scheduleData, shouldShiftWeek = false, weekNumber = getWeekNumber(new Date())) {
+export function getLessonsForDay(targetDay, scheduleData, statusOffset, weekNumber = getWeekNumber(new Date())) {
 	const lessonToday = scheduleData.find(({ day, events }) => day === targetDay && events)
```

Comment:
Why do we even need this `statusOffset` param? 
1. It's name is really confusing and `weekOffset` looks bad too.

As I can see it only adds `+ 1` to the current week number if user passes `{ weekOffset: true }`. This function already has `weekNumber` param to control from which week to take schedule from.

I think we can easily remove this param at all.